src/dreyfus_index_updater.erl

```diff
@@ -64,7 +73,7 @@ update(IndexPid, Index) ->
         couch_db:close(Db)
     end.
 
-load_docs(FDI, _, {I, IndexPid, Db, Proc, Total, LastCommitTime}=Acc) ->
+load_docs(FDI, {I, IndexPid, Db, Proc, Total, LastCommitTime}=Acc) ->
```

Comment:
ok, we can't merge this PR until the pluggable storage engine API is also merged.